SPECS/golang/golang-1.18.spec

```diff
@@ -0,0 +1,247 @@
+%global goroot          %{_libdir}/golang
```

Comment:
Should we not add a `version` constraint on the offending packages.
e.g
`k3s, helm, terraform` ?

Reply:
it is already done in the relevant spec. I assumed that full build would get golang 1.18.8 from PMC but it do not
so I created golang 1.18 spec